src/components/textarea.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(app) {
             }
             return 'formio/components/texteditor.html';
           }
+          else if (!$scope.readOnly && $scope.component.components) {
```

Comment:
It's more than just a boolean.  The array of form components is passed so that it can be used in the ACE editor autocomplete popup.  See the formioScriptEditor directive, lines 36 and 44.  Sorry about the lack of clarifying comments.  Adding another boolean could be seen as redundant.  To avoid confusion I could rename 'components' something like 'autoCompleteComponents'.